.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+## Testing
+
+This app doesn't have CI setup, and it would be misleading to do so:
+the behaviour of the tests changes depending on the environment they
+are run in. **You should manually test your PR before merging.**
+
+- [ ] Tested change in at least one environment
```

Comment:
Nit: To make it easier to read, you can start with the instructions of what the user needs to do, followed by explanation.

```suggestion
**You should manually test your PR before merging.**

- [ ] Tested change in at least one environment

This app doesn't have CI setup, and it would be misleading to do so:
the behaviour of the tests changes depending on the environment they
are run in. 
```

Reply:
Sure, I've rewritten this bit.